ton_client/client/src/tests/mod.rs

```diff
@@ -403,68 +387,63 @@ impl TestClient {
         finished: bool,
     ) {
         //log::debug!("on_result response-type: {} params_json: {}", response_type, params_json);
-        let requests =  &mut TEST_RUNTIME
-            .lock()
-            .await
-            .requests;
-        let request = requests
-            .get_mut(&request_id)
-            .unwrap();
+        let requests = &mut TEST_RUNTIME.lock().await.requests;
+        let request = match requests.get_mut(&request_id) {
+            Some(request) => request,
+            None => return,
+        };
 
-        match ResponseType::from_u32(response_type) {
-            Some(std_response_type) => {
-                match std_response_type {
-                    ResponseType::Success => {
-                        request.sender.send(JsonResponse {
-                            result_json: params_json.to_string(),
-                            error_json: String::new(),
-                        })
-                        .await
-                        .unwrap();
-                    },
-                    ResponseType::Error => {
-                        request.sender.send(JsonResponse {
-                            result_json: String::new(),
-                            error_json: params_json.to_string(),
-                        })
-                        .await
-                        .unwrap();
-                    },
-                    _ => {}
-                };
-            },
-            None => {
-                (request.callback)(params_json.to_string(), response_type).await
-            }
+        if response_type == ResponseType::Success as u32 {
```

Comment:
Add default branch with condition `if response != ResponseType::Nop as u32` and panic to check invalid response types

Reply:
panicked